src/main/java/it/smartcommunitylab/aac/saml/provider/SamlAuthenticationProvider.java

```diff
@@ -29,22 +29,25 @@
 import it.smartcommunitylab.aac.saml.model.SamlUserAccount;
 import it.smartcommunitylab.aac.saml.model.SamlUserAuthenticatedPrincipal;
 import java.io.Serializable;
-import java.util.Collections;
-import java.util.Map;
+import java.util.*;
```

Comment:
replace the wildcard imports with only the specific imports for the classes used in the file